steps/id-and-residency/carer.tsx

```diff
@@ -30,9 +30,9 @@ const step: ProcessStepDefinition = {
   heading: "Carer",
   step: {
     slug: PageSlugs.Carer,
-    nextSlug: PageSlugs.Submit,
+    nextSlug: PageSlugs.Sections,
     Submit: makeSubmit({
-      href: hrefForSlug(PageSlugs.Submit),
+      href: hrefForSlug(PageSlugs.Sections),
```

Comment:
For now, let's send this directly to `PageSlugs.HomeCheck`, to avoid the sections issue. I'll take a look at resolving it.